tests/test_io.py

```diff
@@ -254,3 +255,11 @@ def test_io_datapackage(test_df, tmpdir):
     # read from csv assert that IamDataFrame instances are equal
     import_df = read_datapackage(file)
     assert_iamframe_equal(test_df, import_df)
+
+
+def test_io_netcdf(test_df_year):
+    file = Path(TEST_DATA_DIR / "test_df.nc")
+    import_df = read_netcdf(file)
+    # temporarily fix for np.nan being converted to 'nan' when saving netcdf file
+    import_df.meta.string[1] = np.nan
+    assert_iamframe_equal(test_df_year, import_df)
```

Comment:
Suggest to use `obs` to be consistent with the general obs(erved) vs. exp(ected) structure of tests.
```suggestion
    obs = read_netcdf(TEST_DATA_DIR / "test_df.nc")
    # temporarily fix for np.nan being converted to 'nan' when saving netcdf file
    obs[1] = np.nan
    assert_iamframe_equal(obs, test_df_year)
```